data/splitContent.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "splitContent",
+    "props": {
+      "children": [
+        [
```

Comment:
Yes, I didn't really like the double list. A subtype could work, like with tabs. 

@tyom I don't know if that's something that should be handled as part of this response. The do/don't component could be an example of this and then the UI could apply the correct syntax to the markdown list?